app/assets/stylesheets/announcements.scss

```diff
@@ -0,0 +1,40 @@
+.announcements {
+  display: block;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.content-wrapper {
```

Comment:
Prefix every class in this file with announcements

Reply:
made changes accordingly